packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/aspnet/configorder.md

```diff
@@ -1,15 +1,13 @@
 ## Order of the configuration options
 
-When multiple configurations options are used simultaneously, the SDK chooses
-the option based on the following order:
+When multiple configurations options are used simultaneously, the SDK selects the configuration to use based on the following order:
 
-* SDK Client constructor
-* Environment variables
-* Configuration
+1. SDK Client constructor
+2. Environment variables
+3. Configuration
 
 For example, values set in the SDK Client constructor override the
 environment variables and configuration file settings. Subsequently,
 the environment variables take precedence over the configuration file.
 
-> **Note:** To avoid confusion on how the configuration values are being
-set, we recommend that you only use one configuration option in your solution.
+> **Note:** To avoid confusion with which configuration values that are used by the SDK, you should use only one configuration option in your solution.
```

Comment:
```suggestion
> **Note:** To avoid confusion as to which configuration values are used by the SDK, you should use only one configuration option in your solution.
```